pkg/pillar/agentlog/agentlog_test.go

```diff
@@ -219,3 +234,897 @@ func TestPubsubLog(t *testing.T) {
 	sub.ProcessChange(change)
 	assert.True(t, deleted)
 }
+
+// Memory PSI collector tests
+
+func getSHA256Hash(content []byte) string {
+	hash := sha256.Sum256(content)
+	return hex.EncodeToString(hash[:])
+}
+
+func increaseTimeByASec(timestamp string) string {
+	t, err := time.Parse("2006-01-02 15:04:05", timestamp)
+	if err != nil {
+		panic(err)
+	}
+	t = t.Add(time.Second)
+	return t.Format("2006-01-02 15:04:05")
+}
+
+// Generate a random value in the range [low, high] with a skewed distribution
+// The skewness parameter determines how much the distribution is skewed towards the low end
+// A skewness of 1.0 means no skewness
+func skewedRandomValue(low, high int64, skewness float64) int64 {
+	// Seed the random number generator
+	seed := time.Now().UnixNano()
+	r := rand.New(rand.NewSource(seed))
+
+	// Generate a random float between 0 and 1
+	u := r.Float64()
+
```

Comment:
```suggestion
	u := rand.Float64()
```
What does `u` stand for?

Reply:
unix nano?... Will fix with something more meaningful

Reply:
fixed